lang/en.json

```
@@ -813,6 +813,7 @@
           "FreeManeuver": "Free Maneuver",
           "Triggered": "Triggered Action",
           "FreeTriggered": "Free Triggered Action",
+          "Move": "Move Action",
```

Looks like it doesn't get the "action" label in the book
```suggestion
          "Move": "Move",
```